### ISSUE TITLE
(Enchancement) Ability to refresh the "Last message sent" time.

### ISSUE THREAD

**Issue by Darwish8040** [JustOptimize/return-ShowHiddenChannels#132]
An ability to refresh the last message sent time without having to restart discord would be excellent.

**Comment by JustOptimize** [comment]
As discussed in a previous issue, I don't know whether to implement it or not.

It could get people banned since the plugin would probably have to fetch channels again, but I don't think the discord client does that often if at all.

If anyone has a different solution please send a message in this issue

**Comment by BySwiizen** [comment]
Add a button that could refresh and warn us using a popup for example (it would require confirmation by pressing again to validate the refresh) that there is a potential risk in using this method while waiting for a safe method.

**Comment by Tharki-God** [comment]
still bad and we dont do bad things in this economy

**Comment by angrybirdice** [comment]
yeah, also the zerelibary plugin is unable to be redownloaded

**Comment by JustOptimize** [comment]
> yeah, also the zerelibary plugin is unable to be redownloaded

uh?

**Comment by angrybirdice** [comment]
> > yeah, also the zerelibary plugin is unable to be redownloaded
> 
> uh?
yeah, it shows an error when trying to download that plugin but I think it got fixed for a weird reason since I tried it again it allowed me to download it again. I know you are not the author of it but I was just letting you know that I encountered that issue. hidden channels plugin doesn't work but its already not demanding or anything since you just taking your time.
